bndtools.api/src/org/bndtools/api/BndtoolsConstants.java

```diff
@@ -15,6 +15,7 @@
 
     public static final String MARKER_BND_PROBLEM = "bndtools.builder.bndproblem";
     public static final String MARKER_BND_PATH_PROBLEM = "bndtools.builder.bndpathproblem";
+    public static final String MARKER_COMPONENT = "org.bndtools.builder.componentmarker";
```

Comment:
lets follow the previous convention of using 'bndtools.builder.componentmarker'.  Sorry, you may have had it this way originally.